assets/wizards/readerRevenue/views/donation/index.tsx

```diff
@@ -129,19 +132,34 @@ export const DonationAmounts = () => {
 											<MoneyInput
 												currencySymbol={ currencySymbol }
 												label={ __( 'Low-tier' ) }
-												value={ amounts[ section.key ][ 0 ] }
+												value={
+													amounts[ section.key ][ 0 ] >= minimumDonation
+														? amounts[ section.key ][ 0 ]
+														: minimumDonation
```

Comment:
That's too heavy-handed. This does not allow input of an amount that starts with an integer lower than the minimum. So `12` will be discarded. I think an error message is a better idea than coercing the value to a minimum.

Reply:
Ah, that's an interesting point. Will refactor to show an error message instead.

Reply:
0abcf88 adds warnings instead of coercing the values:

<img width="1068" alt="Screen Shot 2022-08-16 at 10 45 05 AM" src="https://user-images.githubusercontent.com/2230142/184934004-1c3b45c9-4bbb-4b81-9d34-06e246ca7060.png">